wiki/osu!stream/de.md

```diff
@@ -0,0 +1,254 @@
+
+Es unterstützt kein klassisches osu! Beatmaps. Stattdessen werden Karten von einem speziellen Team von Kartierern erstellt, um die neuen Elemente zu nutzen und strengere Standards einzuhalten. Sowohl kostenlose als auch kostenpflichtige Karten sind im "Store" erhältlich.
+
+[osu! stream wurde zusammen mit einem Interview in osu! month \ # 5 ausgearbeitet.](https://osu.ppy.sh/community/forums/topics/59924)
+
+## Installation
+
+![osu! stream in iDevice][Gerät]
+
+### App Store-Methode von iDevice (empfohlen)
+
+**Stellen Sie sicher, dass das iDevice eine Verbindung zum Internet herstellen kann (entweder über Wi-Fi oder 3G / 4G).**
+
+- Gehen Sie vom iDevice zum "App Store"
+- Suchen Sie nach osu! Stream
+- Drücken Sie und installieren Sie osu! Stream
+  - Wenn Sie aufgefordert werden, die Apple ID und das Passwort einzugeben, tun Sie dies bitte.
+- osu! stream kann abgespielt werden, nachdem der Download und die automatische Installation abgeschlossen sind.
+  - Es wird empfohlen, auch den Abschnitt "Hinzufügen von Beatmaps" zu lesen, um die neuesten kostenlosen Songs zu erhalten. Der aktuelle Download enthält nur 2 Songs.
+
+### USB-Methode
+
+** Auf dem Gerät muss [iTunes](https://www.apple.com/itunes/) installiert sein. **
+
+- Gehen Sie zu [diesem Link und laden Sie ihn herunter](http://itunes.apple.com/us/app/osu!stream/id436952197?ls=1&mt=8)
+  - Wenn Sie nach der Apple ID gefragt werden, geben Sie die Apple ID und das Passwort ein.
+- Stecken Sie das iDevice in das Gerät (das über osu! Stream verfügt).
+- Übertragen Sie die osu! -Stream-Dateien mit iTunes.
+- Der iDevice- und der osu! -Stream können abgespielt werden.
+
+## Hinzufügen von Beatmaps
+
+**Stellen Sie sicher, dass das iDevice eine Verbindung zum Internet herstellen kann (entweder über Wi-Fi oder 3G / 4G).**
+
+- Öffnen Sie den osu! -Stream.
+- Drücken Sie die osu! Symbol und es sollte zum Hauptmenü geleitet werden (oder Tutorial, wenn das erste Mal)
+- Drücken Sie die Leiste "Store"
+  - Alternativ gehen Sie zu "Play" und drücken Sie "Download more songs!"
+- Wählen Sie eine gewünschte Packung.
+  - Es wird empfohlen, zuerst Pakete herunterzuladen, die kein Geld kosten. Gratis-Packs haben anstelle des "USD $" -Kosten-Tags "FREE".
+
+## Was ist der Unterschied?
+
+- **Nur auf iOS-Geräten spielbar.** (vorerst)
+- Beatmaps werden von Grund auf neu erstellt.
+- Zum Herunterladen der Song-Packs ist kein Konto erforderlich.
+- Kann osu! Stream mit einem Twitter-Account integrieren.
+- Der Highscore kann anonym unter dem Namen "Gast" an die Site gesendet werden (falls dies vorkommt, jedoch eine Internetverbindung erforderlich ist).
+- Die Kreisuhr wird ausgetauscht und mit einer kleinen Randlinie nach unten verschoben, die je nach Genauigkeit farbig ist. \ [Bottom (Long)\]

```

Comment:
```suggestion
- Die Kreisuhr wird ausgetauscht und mit einer kleinen Randlinie nach unten verschoben, die je nach Genauigkeit farbig ist. \[Bottom (Long)\]
```

The `[` character is being escaped with the `\\` character, there cannot be a space in between the two characters.